uber/templates/regform.html

```diff
@@ -161,17 +161,17 @@
         </div>
         {% popup_link "../static_views/affiliates.html" "What's an affiliate?" %}
     </div>
-{% endif %}
 
```

Comment:
Oops, this was my fault - I didn't realize this was being displayed on the admin form.